addons/im_livechat/controllers/main.py

```diff
@@ -162,7 +162,6 @@ def get_session(self, channel_id, anonymous_name, previous_operator_id=None, cha
             if guest:
                 store.add_global_values(guest_token=guest.sudo()._format_auth_cookie())
         request.env["res.users"].with_context(guest=guest)._init_store_data(store)
-        guest._bus_send_store(store)
```

Comment:
Note: likely unnecessary since chathub localstorage PR, as other tabs will fetch channel info if they don't have it.
Also it feels really strange to have the need to send only on guest but not on user.